diff --git a/node_modules/react-digraph/typings/index.d.ts b/node_modules/react-digraph/typings/index.d.ts
index 904645a..20d07f8 100644
--- a/node_modules/react-digraph/typings/index.d.ts
+++ b/node_modules/react-digraph/typings/index.d.ts
@@ -173,6 +173,8 @@ declare module 'react-digraph' {
     rotateEdgeHandle?: boolean;
     centerNodeOnMove?: boolean;
     initialBBox?: IBBox;
+
+    nodeLocationOverrides: Object | undefined;
   };
 
   export type IGraphInput = {
